lms/djangoapps/certificates/views/webview.py

```diff
@@ -642,8 +643,30 @@ def render_html_view(request, course_id, certificate=None):
         # Track certificate view events
         _track_certificate_events(request, course, user, user_certificate)
 
+        try:
+            # .. filter_implemented_name: CertificateRenderStarted
+            # .. filter_type: org.openedx.learning.certificate.render.started.v1
+            context, custom_template = CertificateRenderStarted.run_filter(
+                context=context,
+                custom_template=custom_template,
+            )
+        except CertificateRenderStarted.RenderAlternativeCertificate as exc:
+            response = _render_invalid_certificate(
```

Comment:
looking at this, do you think that the control should necessary mean that if a filter developer wants to render an alternative template, this must mean that it is invalid?
Or is the case that if the template is valid, then I just answer the filter with a value for `custom_template` and don't raise anything?

Reply:
>  is the case that if the template is valid, then I just answer the filter with a value for custom_template and don't raise anything?

That it's what I've been thinking. Should we change the name to `RenderInvalidCertificate`? Should we have these two possibilities in the other PR(s)? isn't that confusing? :thinking:

Reply:
Probably the most correct would be then `RenderAlternativeInvalidCertificate` as the exeption and leave the custom-template with no exceptions as the way to render a valid certificate with a different path

Reply:
Done! :partying_face: